### PR TITLE
Feat-503 create operational cards list

### PR DESCRIPTION
**Pull request by 9Syncopation**
as per suggested steps in the issue 
left most of the widgets within operationalCards property  on comments to see behavior

### CODE REVIEW

**Comment by yuvalbl** [comment]
please add task number to PR name

File: src/services/widgets.data.service.ts [hunk 1]
```diff
@@ -31,15 +32,22 @@ export const fetchWidgets = async (id: number, yearAgo?: number): Promise<ILocat
 };
 
 function processWidgetsFetchResponse(response: any) {
-  console.log(response);
   const result = {
     meta: response.data.meta,
-    widgets: getVerifiedWidgetsData(response.data.widgets),
+    widgets: response.data.widgets,
```
Comment (yuvalbl) [inline src/services/widgets.data.service.ts:37]: null?

File: src/services/widgets.data.service.ts [hunk 2]
```diff
@@ -31,15 +32,22 @@ export const fetchWidgets = async (id: number, yearAgo?: number): Promise<ILocat
...
+  if (showOnlyOperCards) {
+    result.widgets = getOperWidgetData(response.data.widgets);
+  }
+  result.widgets = getVerifiedWidgetsData(result.widgets);
   result.widgets = addWidgetsVariants(result.widgets);
   return result;
 }
 
+function getOperWidgetData(widgets: Array<any>) {
+  const widgetsToShow = widgets.filter((widget) => operationalCards.includes(widget.name));
```
Comment (yuvalbl) [inline src/services/widgets.data.service.ts:48]: can be returned in the same line
`return widgets.filter((widget) => operationalCards.includes(widget.name));`

Comment (9Syncopation) [reply src/services/widgets.data.service.ts]: fixed

File: src/services/widgets.data.service.ts (hunk 1 above)
Comment (9Syncopation) [reply src/services/widgets.data.service.ts]: refactor  to>>
`widgets = getVerifiedWidgetsData(widgets);
  widgets = addWidgetsVariants(widgets);`

please note: it is different than the suggested solution at  issue 
https://github.com/hasadna/anyway-newsflash-infographics/issues/503

 ` widgets = getVerifiedWidgetsData(response.data.widgets);
  widgets = addWidgetsVariants(response.data.widgets);`

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**